storage/redis_test.go

```diff
@@ -43,28 +44,6 @@ func TestRedisAddressConfiguration(t *testing.T) {
 		}
 	})
 
-	t.Run("Default addresses", func(t *testing.T) {
-		opts := &redis.UniversalOptions{}
-		simpleOpts := opts.Simple()
-
-		if simpleOpts.Addr != "127.0.0.1:6379" {
-			t.Fatal("Wrong default single node address")
-		}
-
-		opts.Addrs = []string{}
-		clusterOpts := opts.Cluster()
-
-		if clusterOpts.Addrs[0] != "127.0.0.1:6379" || len(clusterOpts.Addrs) != 1 {
-			t.Fatal("Wrong default cluster mode address")
-		}
-
-		opts.Addrs = []string{}
-		failoverOpts := opts.Failover()
-
-		if failoverOpts.SentinelAddrs[0] != "127.0.0.1:26379" || len(failoverOpts.SentinelAddrs) != 1 {
-			t.Fatal("Wrong default sentinel mode address")
-		}
-	})
```

Comment:
This test has been removed since it was testing Redis library-specific stuff - not related to the storage library